boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -580,4 +648,150 @@ TEST_F(
         0.0f, App_CanTx_GetPeriodicSignal_TORQUE_REQUEST(can_tx_interface));
 }
 
+// DCM21
+TEST_F(DcmStateMachineTest, inverter_switches_closed_when_AIRS_closed)
+{
+    // Both inverter switches are open to begin with
+    LetTimePass(state_machine, 10);
+    ASSERT_FALSE(App_InverterSwitches_IsRightOn(inverter_switches));
+    ASSERT_FALSE(App_InverterSwitches_IsLeftOn(inverter_switches));
+    ASSERT_EQ(turn_on_right_inverter_fake.call_count, 0);
+    ASSERT_EQ(turn_on_left_inverter_fake.call_count, 0);
+
+    // Close AIR-, both inverter switches should still be open
+    App_CanRx_BMS_AIR_STATES_SetSignal_AIR_NEGATIVE(
+        can_rx_interface, CANMSGS_BMS_AIR_STATES_AIR_NEGATIVE_CLOSED_CHOICE);
+    LetTimePass(state_machine, 10);
+    ASSERT_FALSE(App_InverterSwitches_IsRightOn(inverter_switches));
+    ASSERT_FALSE(App_InverterSwitches_IsLeftOn(inverter_switches));
+    ASSERT_EQ(turn_on_right_inverter_fake.call_count, 0);
+    ASSERT_EQ(turn_on_left_inverter_fake.call_count, 0);
+
+    // Close AIR+, both inverter switches should now be closed
+    App_CanRx_BMS_AIR_STATES_SetSignal_AIR_POSITIVE(
+        can_rx_interface, CANMSGS_BMS_AIR_STATES_AIR_POSITIVE_CLOSED_CHOICE);
+    LetTimePass(state_machine, 10);
+    ASSERT_TRUE(App_InverterSwitches_IsRightOn(inverter_switches));
+    ASSERT_TRUE(App_InverterSwitches_IsLeftOn(inverter_switches));
+    ASSERT_EQ(turn_on_right_inverter_fake.call_count, 1);
+    ASSERT_EQ(turn_on_left_inverter_fake.call_count, 1);
+}
+
+TEST_F(DcmStateMachineTest, init_to_fault_state_on_left_inverter_fault)
+{
+    LetTimePass(state_machine, 10);
```

Comment:
Why LetTimePass here?

Reply:
same for line 721

Reply:
noticed a couple more places where you LetTimePass before checking that we are in the init state. Is this necessary?

Reply:
No, it is not necessary. I'll take them out

Reply:
fixed